modules/core/src/webgl/classes/program.js

```diff
@@ -98,37 +98,25 @@ export default class Program extends Resource {
 
   // A good thing about the WebGL API is that there are so many ways to draw things ;)
   // This function unifies those ways into a single call using common parameters with sane defaults
-  draw({
```

Comment:
draw function is getting really big, splitting it in pieces, maybe should move to separate file